pcrelate.R

```diff
@@ -0,0 +1,59 @@
+#! /usr/bin/env Rscript
+library(argparser)
+library(magrittr)
+
+st_help <- "Threshold for sparsifying GRM (will be multiplied by scale_kin)"
+argp <- arg_parser("Run PC-Relate") %>%
+  add_argument("gds_file", help = "GDS file") %>%
+  add_argument("pcair_file", help = "File with pcair object") %>%
+  add_argument("--out_prefix", help = "Prefix for output files",
+               default = "") %>%
+  add_argument("--n_pcs", default = 10,
+               "Number of PCs to pass to pcrelate") %>%
+  add_argument("--variant_id", help = "File with vector of variant IDs") %>%
+  add_argument("--sample_id", help = "File with vector of sample IDs") %>%
+  add_argument("--scale_kin", help = "Scaling factor for GRM output",
+               default = 1) %>%
+  add_argument("--sparse_thresh", help = st_help, default = 0) %>%
+  add_argument("--scan_block_size", help "pcrelate scan block size",
+               default = 5000) %>%
+  add_argument("--correct", "Flag to implement small-sample correction",
```

Comment:
The way this is set up is a little funny. The default function argument is `correct = TRUE`, which is _not_ the default for this script. The reason for that is that `argv$correct == TRUE` if you use `--correct` from the command line, and it is `FALSE` otherwise. The reason I set it up that way is because I think it's not usually the way command line flags are set up to use `--flag` to specify something as `FALSE`

Reply:
The default in `pcrelate` is actually `small.samp.correct = FALSE`, so having the `--correct` flag to change the default is actually perfect. But I think you should call it `--small_samp_correct` to make it clear what the correction is.

Reply:
Ah, that's what I get for reading the man page online rather than the one from within R for the specific version of a package. I was going off of [this help page](https://www.rdocumentation.org/packages/GENESIS/versions/2.2.2/topics/pcrelate). I'll make the change